packages/web3-validator/src/validator.ts

```diff
@@ -67,6 +67,10 @@ const convertToZod = (schema: JsonSchema): ZodType => {
 	}
 
 	if (schema?.format) {
+		if (!formats[schema.format]) {
+			throw new Error(`format ${schema.format} is unsupported`);
```

Comment:
I re invoked tests and not unit tests are passing, but codecov patch is not, could you add unit test hitting L71, Thanks @fullkomnun